examples/with-graphql-hasura/README.md

```diff
@@ -0,0 +1,34 @@
+# Hasura GraphQL Starter Example 
+
+This example shows you how to use [Hasura's self-hosted GraphQL Engine](https://github.com/hasura/graphql-engine/) in your Next.js project.
```

Comment:
Could we make this example use the managed GraphQL engine Hasura provides instead? I think that's an easier starting point for folks.